ddtrace/internal/processor/trace.py

```diff
@@ -309,3 +313,10 @@ def on_span_finish(self, span):
                 span.set_tag_str(self._config.tag_name, peer_service_definition)
                 span.set_tag_str(self._config.source_tag_name, data_source)
                 return
+
+    def _remap_peer_service(self, span):
+        current_peer_service = span.get_tag(self._config.tag_name)
+
+        if current_peer_service in self._config.peer_service_mapping:
+            span.set_tag_str(self._config.remap_tag_name, current_peer_service)
```

Comment:
Just to make sure, we want to set the "remapped_from" tag to the prior value of `peer.service` and not `peer.service.source`, correct?

Reply:
right. If something maps (hence you replaced values) this will hold the old value